reviews/incubation-crossplane.md

```diff
@@ -0,0 +1,288 @@
+
+### Sandbox Acceptance
+
+Crossplane was accepted into the CNCF Sandbox on June 24th, 2020.
+
+## Incubation Proposal Resources
+
+* Due Diligence Document: [DD doc](https://docs.google.com/document/d/1OBGR7h7mqQ1zE1yheIH8oPkTHiVaEH_DdT6QCTdF370/edit?usp=sharing)
+* SIG App-Delivery presentation: [Slide deck](https://docs.google.com/presentation/d/12YLgn_J5WyvW7s4dab8eF0ebquWzB4Zv3bzi-klUW1w/edit?usp=sharing)
+
+## Progress since joining the CNCF Sandbox
+
+The biggest accomplishment from the Crossplane community is the release of [v1.0, the first major
+milestone](https://blog.crossplane.io/announcing-crossplane-v1-0/) in the project’s history, which
+was followed recently by even more improvements in
+[v1.1](https://blog.crossplane.io/crossplane-v1-1-vault-integration-enhanced-composition-and-aws-provider-code-generation/).
+With these versions of Crossplane, the project is now stable and ready to be used in production
+scenarios. This is further validated by project adopters successfully running Crossplane in
+production environments. The core APIs have been moved to v1 (stable) and future enhancements and
+improvements are intended to be done in a backwards compatible way with a clear upgrade path.  We do
+not anticipate any further breaking changes in subsequent minor and patch releases.
+
+With Crossplane reaching maturity, we are seeing end-users adopt Crossplane and find success running
+it in their production environments. We consistently hear from users that have built their own
+custom platforms APIs to enable self-service provisioning for their developers and accelerate the
+delivery of their software to production. The community growth has been quite encouraging and we are
+pleased to be able to include specific details about some of their adoption success in the sections
+below.
+
+The Crossplane community has expanded its collaboration with a rich ecosystem of partners, building
+relationships and combining development efforts with a broader set of cloud providers and projects
+in meaningful ways:
+
+* IBM developed and released the [first version of their Crossplane provider for IBM
+  Cloud](https://developer.ibm.com/blogs/ibm-joins-the-crossplane-community/), enabling the
+  provisioning and management of 85+ hosted services from the IBM Cloud Catalog.
+* Engineering teams from both [AWS and
+  Azure](https://blog.crossplane.io/accelerating-crossplane-provider-coverage-with-ack-and-azure-code-generation-towards-100-percent-coverage-of-all-cloud-services/)
+  have joined forces with the Crossplane community to share their code generation pipelines,
+  allowing the Crossplane providers for AWS and Azure to increase their coverage and more easily
+  stay updated with changes in their cloud offerings.
+* RedHat partners with Crossplane for infrastructure provisioning, with a focus on hybrid scenarios.
+  Crossplane has been available on [operatorhub.io](https://operatorhub.io/operator/crossplane)
```

Comment:
I can't see this anymore https://operatorhub.io/operator/crossplane . Not sure if it was replace with https://operatorhub.io/operator/universal-crossplane